samples/Azure.ResourceManager.Sample/Generated/LongRunningOperation/SampleArmOperation.cs

```diff
@@ -43,6 +46,9 @@ internal SampleArmOperation(ClientDiagnostics clientDiagnostics, HttpPipeline pi
         public override string Id => throw new NotImplementedException();
 #pragma warning restore CA1822
 
+        /// <inheritdoc />
+        public override RehydrationToken? GetRehydrationToken() => _rehydrationToken;
```

Comment:
I don't think you can store this and will need to calculate it every time since each time Update is called the value can change

Reply:
Actually, we don't have a way to re-calculate it within `ArmOperation`. We don't get the instance of `NextLinkOperationImplementation` in place, we only get internal `IOperation` back, which we don't want to add `GetRehdyrationToken()`.
It's OK to have the old token, the drawback of using it is, it will cause extra http traffic forwarding to latest polling URL.

Reply:
never mind, we can cast it back and re-calculate rehydration token.

Reply:
Update to get latest calculation for incomplete operation, get previous cache for complete operation.